repos/fdbt-site/src/design/modules/_components.scss

```diff
@@ -135,3 +135,7 @@
         color: govuk-shade(govuk-colour('red'), 20%);
     }
 }
+
+.remove-top-margin {
```

Comment:
There is a gov-uk margin 0 style already

Reply:
Ah yeah my bad.